ddtrace/contrib/botocore/services/bedrock.py

```diff
@@ -0,0 +1,292 @@
+        }
+    elif provider == "anthropic":
+        return {
+            "prompt": request_body.get("prompt"),
+            "temperature": request_body.get("temperature", None),
+            "top_p": request_body.get("top_p", None),
+            "top_k": request_body.get("top_k", None),
+            "max_tokens": request_body.get("max_tokens_to_sample", None),
+            "stop_sequences": request_body.get("stop_sequences", []),
+        }
+    elif provider == "cohere":
+        return {
+            "prompt": request_body.get("prompt"),
+            "temperature": request_body.get("temperature", None),
+            "top_p": request_body.get("p", None),
+            "top_k": request_body.get("k", None),
+            "max_tokens": request_body.get("max_tokens", None),
+            "stop_sequences": request_body.get("stop_sequences", []),
+            "stream": request_body.get("stream", None),
+            "n": request_body.get("num_generations", None),
+        }
+    elif provider == "meta":
+        return {
+            "prompt": request_body.get("prompt"),
+            "temperature": request_body.get("temperature", None),
+            "top_p": request_body.get("top_p", None),
+            "max_tokens": request_body.get("max_gen_len", None),
+        }
+    elif provider == "stability":
+        # TODO: request/response formats are different for image-based models. Defer for now
+        return {}
+    return {}
+
+
+def _extract_response(span: Span, body: Dict[str, Any]) -> Dict[str, List[str]]:
+    """
+    Extracts text and finish_reason from the response body, which has different formats for different providers.
+    """
+    text, finish_reason = None, None
+    provider = span.get_tag("bedrock.request.model_provider")
+    if provider == "ai21":
+        text = body.get("completions")[0].get("data").get("text")
+        finish_reason = body.get("completions")[0].get("finishReason")
+    elif provider == "amazon":
+        text = body.get("results")[0].get("outputText")
+        finish_reason = body.get("results")[0].get("completionReason")
+    elif provider == "anthropic":
+        text = body.get("completion")
+        finish_reason = body.get("stop_reason")
+    elif provider == "cohere":
+        text = [generation["text"] for generation in body.get("generations")]
+        finish_reason = [generation["finish_reason"] for generation in body.get("generations")]
+        for i in range(len(text)):
+            span.set_tag_str("bedrock.response.choices.%d.id" % i, str(body.get("generations")[i]["id"]))
+    elif provider == "meta":
+        text = body.get("generation")
+        finish_reason = body.get("stop_reason")
+    elif provider == "stability":
+        # TODO: request/response formats are different for image-based models. Defer for now
+        pass
+
+    if not isinstance(text, list):
+        text = [text]
+        finish_reason = [finish_reason]
```

Comment:
Should this also be turned into a list with the condition that it already isn't one, as is done in `_extract_streamed_response`? Looks like in the case of `cohere` it might already be a list.

Reply:
Good catch! Will update.